src/app/reports/reportsTable/ReportsPage.tsx

```diff
@@ -0,0 +1,130 @@
+"use client";
```

Comment:
Most of this code is based off the way ClientsPage was implemented. Same with the getReportsData.